### PR TITLE
Fix issue with date of retrieved messages.

### PR DESCRIPTION
**Pull request by mayank408**
Fixes issue #697 
Now the date of the retrieved messages is set according to the query date coming from the response from the server.

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java [hunk 1]
```diff
@@ -1070,20 +1091,20 @@ public void onFailure(Call<SusiResponse> call, Throwable t) {
         }
     }
 
-    private void updateDatabase(final long prevId, final String message, final boolean isDate, final boolean mine,
+    private void updateDatabase(final long prevId, final String message, final boolean isDate, final String Date, final boolean mine,
```
Comment (betterclever) [inline app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java]: Please strictly follow variable naming convention. Variable name starts with lowercase.

**Comment by mayank408** [comment]
@betterclever The timezone is local.

**Comment by betterclever** [comment]
@mayank408 So, server sends date and time in local timezone. Right?

**Comment by mayank408** [comment]
@betterclever We send timezone at the time of query which is saved in server. So yes It is local

**Comment by betterclever** [comment]
@mayank408 Okay.

**Pull request merged**